src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -11,7 +16,7 @@ import Badge from '../../Common/Badge';
 import { CacheItem } from '../../../../server/interfaces/api/settingsInterfaces';
 import { formatBytes } from '../../../utils/numberHelpers';
 
-const messages = defineMessages({
+const messages: { [messageName: string]: MessageDescriptor } = defineMessages({
```

Comment:
Why are we typing this ourselves? `defineMessages` should already be returning the correct type?

Reply:
It doesn't like it when you do `messages[someString]` :(